apps/dolly-frontend/src/main/js/src/pages/minSide/maler/Maloversikt.tsx

```diff
@@ -37,7 +39,13 @@ export default ({ brukernavn }: { brukernavn: string }) => {
 			dataField: 'malNavn',
 			formatter: (_cell: any, row: { id: string; malNavn: string }) =>
 				erUnderRedigering(row.id) ? (
-					<EndreMalnavn malInfo={row} avbrytRedigering={avbrytRedigering} />
+					<EndreMalnavn
+						malInfo={row}
+						avbrytRedigering={(id: string) => {
+							avbrytRedigering(id)
+							mutate()
```

Comment:
Denne mutate trengs vel ikke, siden den ikke fører til noen endringer i bestillinger når man avbryter? 😅

Reply:
Tror det er litt misvisende navn. Denne blir kun kalt etter at navnet er endret for å avslutte redigeringen (derfor mutate også). Kun "avbrytRedigering" blir brukt på selve avbryt knappen (se lenger ned) :)